### PR TITLE
Moving ui-brand-heha to brands

### PR DESCRIPTION
**Pull request by jackdcrawford**
#### What does this PR do? (please provide any background)
Moves the latest [ui-brand-heha](https://github.com/holidayextras/ui-brand-heha) to this repository to match the uxui roadmap.

#### What tests does this PR have?
None

#### How can this be tested?
- Pull latest `master` on [pattern-library](https://github.com/holidayextras/pattern-library) & pull this branch of brands.
- In the `pattern-library` directory, `npm link ../brands`
- Change `pattern-library/less/heha-theme:72` HEHA Theme section to be

```less
//
// HEHA Theme
// --------------------------------------------------

@import '../node_modules/brands/brands/heha/src/less/theme';

@asset-path: "https://d2r3bolva79zkl.cloudfront.net/assets/";
@asset-image-path: @asset-path;
@asset-font-path: "@{asset-path}fonts";
```

- `npm start` in pattern library & check out the planner. [http://localhost:3000/pages/planner/index.html](http://localhost:3000/pages/planner/index.html) Make sure you have switched to the HEHA! brand in the side menu.

#### Screenshots / Screencast
planner

#### What gif best describes how you feel about this work?
removal van

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by dancork** [comment]
taking :one:

File: brands/heha/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/holidayextras/brands.git"
+  },
+  "author": "Nick Patrick",
```
Comment (dancork) [inline brands/heha/package.json]: ?

File: brands/heha/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/holidayextras/brands/issues"
+  },
+  "homepage": "https://github.com/holidayextras/brands#readme"
```
Comment (dancork) [inline brands/heha/package.json]: should this link directly to the heha readme?

**Comment by dancork** [comment]
image

:+1:

**Comment by jodiedoubleday** [comment]
taking :v:

**Comment by jodiedoubleday** [comment]
:shipit:

**Pull request merged**